recipes/datafusion/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "datafusion" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f98aa882d78a7cd86d23641b557bffe422169b73dc387eb9dddc0e366e5970bf
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - rust
+  host:
+    - python
+    - maturin
+    - pip
+  run:
+    - python
+    - numpy
+    - pandas
+    - pyarrow
+
+test:
+  imports:
+    - datafusion
+
+about:
+  home: https://github.com/apache/arrow-datafusion
+  license: Apache-2.0
+  license_family: APACHE
+  license_file: LICENSE.txt
```

Comment:
You need to generate a second license file using `cargo-bundle-licenses` that bundle the statically linked lib's licences, see https://github.com/conda-forge/cargo-bundle-licenses-feedstock/blob/18cd199df604fee2f36ca6748a7feb29fc754cf7/recipe/build.sh#L14

Reply:
This seems like the responsibility of the upstream package, no?

Reply:
Also, is there some documentation on why this is necessary? Do all rust packages on conda-forge currently conform to this practice?

Reply:
Not sure, you don't need that in a source distribution, only when you ship static binaries. We've been adding that to all rust packages, haven't seen one yet that does it in upstream :(

Reply:
> Also, is there some documentation on why this is necessary? Do all rust packages on conda-forge currently conform to this practice?

Also not sure whether we have that documented somewhere but all rust (and Go, Haskell, …) packages should conform to this practice. We have been enforcing it only in the past 12months, so older ones likely miss it. 

Here's also an example line where `cargo-bundle-licenses` is installed as a conda package: https://github.com/conda-forge/ripgrep-feedstock/blob/ac840d94244c5a7293e71015df9913944ed0ad7b/recipe/build.sh#L6

Reply:
I don't think there are any binaries here, it's a library.

Reply:
That also a binary ;) "binary" as a synoym for "compiled-and-statically-linked code", not executable.

Reply:
Can you elaborate on (or link to) why this is necessary? Is there a legal reason, or something else?

Reply:
See https://github.com/conda-forge/conda-forge.github.io/issues/1052 for more context on this.